src/data/publicHousingData.ts

```diff
@@ -0,0 +1,216 @@
+
+type publicHousingContact = {
+  area: string;
+  address: string;
+  city: string;
+  phone: string;
+}
+const associations:publicHousingContact[] = [
+ {
+   area: "Anacortes",
+   address: "719 Q Ave",
+   city: "Anacortes, WA 98221",
+   phone: "(360) 293-7831"
+ },
+ {
+   area: "Asotin County",
+   address: "1212 Fair St",
+   city: "Clarkston, WA 99403",
+   phone: "(509) 758-5751",
+ },
+ {
+  area: "Bellingham",
+  address: "208 Unity St",
+  city: "Bellingham, WA 98225",
+  phone: "(360) 676-6887",
+},
+{
+  area: "Bremerton",
+  address: "600 Park Ave,",
+  city: "Bremerton, WA 98337",
+  phone: "(360) 479-3694",
+},
+{
+  area: "Everett Housing Authority",
+  address: "3107 Colby Ave",
+  city: "Everett, WA 98201",
+  phone: "(425) 303-1102",
+},
+{
+  area: "Grant County",
+  address: "1139 Larson Blvd",
+  city: "Moses Lake, WA 98837",
+  phone: "(509) 762-2202",
+},
+{
+  area: "Grays Harbor",
+  address: "602 East First St",
+  city: "Aberdeen, WA 98520",
+  phone: "(360) 532-0570",
+},
+{
+  area: "Island County",
+  address: "7 NW 6th St",
+  city: "Coupeville, WA 98239",
+  phone: "(360) 678-4181",
+},
+{
+  area: "Kalama",
+  address: "226 Cloverdale Rd",
+  city: "Kalama, WA 98625",
+  phone: "(360) 673-2873",
+},
+{
+  area: "Kelso",
+  address: "1415 S 10th",
+  city: "Kelso, WA 98626",
+  phone: "(360) 423-3490",
+},
+{
+  area: "Kennewick",
+  address: "1915 W 4th Pl",
+  city: "Kennewick, WA 99336",
+  phone: "(509) 586-8576",
+},
+{
+  area: "King County",
+  address: "600 Andover Park W",
+  city: "Seattle, WA 98188",
+  phone: "",
```

Comment:
Was there no phone provided? On their site, looks like this: (206) 574‑1100 is their number?

Reply:
> Was there no phone provided? On their site, looks like this: (206) 574‑1100 is their number?
It wasn't in the data we had in figma, I forgot to check back to correct it. Good catch.